src/ddspdrum/module.py

```diff
@@ -353,10 +350,14 @@ def __init__(
         sustain_duration: float,
         pitch_adsr: ADSR = ADSR(),
         amp_adsr: ADSR = ADSR(),
-        vco: VCO = VCO(),
+        vco: Union[VCO, List[VCO]] = VCO(),
         vca: VCA = VCA(),
+        noise_ratio: float = 0
     ):
-        super().__init__(modules=[pitch_adsr, amp_adsr, vco, vca])
+        if type(vco) is not list:
```

Comment:
Then we can remove this, which is not considered very pythonic

Reply:
Yeah it's gross.